dlang.org.ddoc

```diff
@@ -436,12 +435,14 @@ SUBNAV_HEAD=
 SUBNAV_TEMPLATE=$(DIVC subnav-helper) $(DIVC subnav, $0)
 _=
 
-SUBNAV_TOOLS=
+SUBNAV_CLI_REFERENCE=
 $(SUBNAV_TEMPLATE
-    $(SUBNAV_HEAD D-Specific Tools, $(ROOT_DIR)tools.html, overview)
+    $(SUBNAV_HEAD Command-line Reference, http://wiki.dlang.org/Development_tools, overview)
```

Comment:
The subnav macro is rather strict, I would need to break it and I am not sure whether that's worth it, e.g. the macro ensures that the submenu is responsive too.

![image](https://cloud.githubusercontent.com/assets/4370550/15982463/5fc71a24-2f88-11e6-8d3f-b59c51ab8afa.png)